src/app/pages/users-admin/users-admin.service.ts

```diff
@@ -10,26 +11,44 @@ import { environment } from 'src/environments/environment';
   providedIn: 'root',
 })
 export class UsersAdminService {
-  constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient, private auth: AuthenticationService) {}
 
   getUnassignedTests(): Observable<Admin[]> {
+    let header = new HttpHeaders().set(
+      'Authorization',
+      `Bearer ${this.auth.token[1]}`
+    );
     return this.http.get<Admin[]>(
```

Comment:
As far as i can see, you've already have _AuthInterceptor_ that adds Authorization header to every request to the server, so you don't have to duplicate it